src/java.base/share/classes/java/io/RandomAccessFile.java

```diff
@@ -101,8 +101,12 @@ public class RandomAccessFile implements DataOutput, DataInput, Closeable {
 
     /**
      * Creates a random access file stream to read from, and optionally
-     * to write to, a file with the specified pathname. A new
-     * {@link FileDescriptor} object is created to represent the
+     * to write to, a file with the specified pathname.
+     * {@linkplain java.nio.file##links Symbolic links}
+     * are automatically redirected to the <i>target</i> of the link.
+     * If the access mode specifies writing and the file or link target exists,
+     * then it will be truncated.
```

Comment:
I think you've copied the text from FOS but it doesn't work here as RAF doesn't truncate.

For "r" mode, it opens an existing file. For the "rw" and variants, it opens an existing file, or creates a new file if it doesn't exist.

Reply:
Fixed in a6c2168.